HTML/mockup-ellipsis/style.css

```diff
@@ -0,0 +1,458 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+html {
+    /* 1rem = 10px  10px/16px = 0.625 = 62.5% */
+    font-size: 62.5%;
+}
+
+@media only screen and (max-width: 1100px) {
+    html {
+        font-size: 57%;
+    }
+}
+
+@media only screen and (max-width: 500px) {
+    html {
+        font-size: 50%;
+    }
+}
+
+/***** HEADER *****/
+
+header {
+    padding-top: 13%;
+    width: 100%;
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    background-image: url(img/layer-8@3x-min.png);
+    background-size: 100% 100%;
+}
+
+@media only screen and (max-width: 1100px) {
+    header {
+        padding-top: 10%;
+        height: 70vh;
+        background-size: cover;
+        background-position: center;
+    }
+}
+
+@media only screen and (max-width: 500px) {
+    header {
+        padding-top: 7%;
+        height: 50vh;
+    }
+}
+
+.header__container {
+    width: 50%;
+    position: relative;
+    text-align: center;
+}
+
+@media only screen and (max-width: 1100px) {
+    .header__container {
+        width: 60%;
+    }
+}
+
+@media only screen and (max-width: 800px) {
+    .header__container {
+        width: 80%;
+    }
+}
+
+@media only screen and (max-width: 500px) {
+    .header__container {
+        width: 90%;
+    }
+}
+
+picture {
+    display: block;
+}
+
+.header__logo {
+    width: 6.5rem;
+    height: 6.9rem;
```

Comment:
setting both width and height for image may distort the one. Leave only one dimension